docs/guidelines/marketing_guide_to_firefox_l10n.md

```diff
@@ -0,0 +1,164 @@
+# Firefox localization copy guide
+
+This document is intended to facilitate the creation and localization of copy, and to support optimal preparation and consistency. This is a living document and it reflects our current state. As our brand expression continues to evolve, we&#39;ll make updates to this doc.
+
+**Why is this guide important for the localization of Firefox marketing copy?**
+
+This guide is intended to help you better understand Firefox as a brand to get a better sense of how to approach translating in your language. In addition, we&#39;ll share fixed, company-specific word choices and spelling to facilitate the localization process.
```

Comment:
Convert `&#39;` to ’ across the document

Reply:
Done